indigo/bulk_creator.py

```diff
@@ -530,7 +530,7 @@ def link_amendment_active(self, row):
                                  amendment=amendment)
 
     def link_taxonomy(self, row):
-        topics = [x.strip(',') for x in row.taxonomy.split()]
+        topics = [x.strip(',') for x in row.taxonomy.split(', ')]
```

Comment:
Split on `,` only so that `foo,bar` isn't ignored, and then trim. After splitting on a comma, the comma won't appear anywhere in the string.

```
>>> "foo,bar ,  baz  , boom,,, a".split(",")
['foo', 'bar ', '  baz  ', ' boom', '', '', ' a']
>>> [x.strip() for x in "foo,bar ,  baz  , boom,,, a".split(",") if x.strip()]
['foo', 'bar', 'baz', 'boom', 'a']
```